src/mnelab/dialogs/channel_stats.py

```diff
@@ -0,0 +1,95 @@
+# © MNELAB developers
+#
+# License: BSD (3-clause)
+from PySide6.QtCore import Qt
+from PySide6.QtWidgets import (
+    QDialog,
+    QPushButton,
+    QTableWidget,
+    QTableWidgetItem,
+    QVBoxLayout,
+)
+
+from mnelab.utils.channelstats_calculations import calculate_channel_stats
+
+
+class SortableTableWidgetItem(QTableWidgetItem):
+    def __init__(self, value):
+        super().__init__(str(value))
+        self.value = value
+
+    def __lt__(self, other):
+        try:
+            return float(self.value) < float(other.value)
+        except ValueError:
+            return str(self.value) < str(other.value)
+
+
+class ChannelStats(QDialog):
+    def __init__(self, parent, raw):
+        super().__init__(parent=parent)
+
+        # window
+        self.setWindowTitle("Channel Stats")
+        self.resize(650, 550)
```

Comment:
It would be nice if the default size of the dialog was wide enough to avoid showing the horizontal scrollbar.